filemgr/src/main/java/org/apache/oodt/cas/filemgr/cli/action/FileManagerCliAction.java

```diff
@@ -16,52 +16,47 @@
  */
 package org.apache.oodt.cas.filemgr.cli.action;
 
-//JDK imports
 
 import org.apache.commons.lang.Validate;
 import org.apache.oodt.cas.cli.action.CmdLineAction;
 import org.apache.oodt.cas.filemgr.structs.exceptions.ConnectionException;
 import org.apache.oodt.cas.filemgr.system.FileManagerClient;
 import org.apache.oodt.cas.filemgr.util.RpcCommunicationFactory;
 
-import java.io.IOException;
 import java.net.MalformedURLException;
 import java.net.URL;
 
-//Apache imports
-//OODT imports
-
 /**
  * Base {@link CmdLineAction} for File Manager.
  *
  * @author bfoster (Brian Foster)
  */
 public abstract class FileManagerCliAction extends CmdLineAction {
 
-   private FileManagerClient client;
+    private FileManagerClient fmc;
 
-   public String getUrl() {
-      return System.getProperty("org.apache.oodt.cas.filemgr.url");
-   }
+    public String getUrl() {
+        return System.getProperty("org.apache.oodt.cas.filemgr.url");
+    }
 
-   protected FileManagerClient getClient() throws MalformedURLException, ConnectionException {
-      Validate.notNull(getUrl(), "Must specify url");
+    /**
+     * TODO(imesha) Fix client closing problem which makes it usable only once
```

Comment:
should we open up a separate issue for this?

Reply:
Actually this problem is still there. Let it be for now. The problem is, we need to remove the `fmc` attribute of `FileManagerCLIAction`, but cannot be done since the tests are using it to add a mock filemanager client.